changelog/3993.bugfix.rst

```diff
@@ -0,0 +1 @@
+optimised response_block_properties of vso and also fixed its bug of removing None from set without checking if None is present in it or not.
```

Comment:
Updated `VSOClient.response_block_properties`  to check if "None" is in  the return.